client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-top-header-buttons/index.tsx

```diff
@@ -72,6 +76,26 @@ export default function SiteTopHeaderButtons() {
 							)
 						}
 					/>
+					{ shouldRenderAddSitesTour && (
+						<GuidedTour
+							className="jetpack-cloud-site-dashboard__guided-tour"
+							preferenceName="jetpack-cloud-site-dashboard-add-new-site-tour"
+							tours={ [
+								{
+									target: '#sites-overview-add-sites-button .split-button__toggle',
+									popoverPosition: 'bottom left',
+									title: translate( 'Press on an arrow button' ),
+									description:
+										translate( 'Click the arrow button and select "Connect a site to Jetpack".' ) +
+										'\n\n' +
+										translate(
+											'Sites with jetpack installed will automatically appear in the site management view.'
+										),
```

Comment:
Perhaps this instead, coupled with a type change for `description`:

```suggestion
									description: (
										<>
											{ translate(
												'Click the arrow button and select "Connect a site to Jetpack".'
											) }
											<br />
											<br />
											{ translate(
												'Sites with jetpack installed will automatically appear in the site management view.'
											) }
										</>
									),
```

Reply:
Change included here: 4979eeac816e26b4b3989a8a9798c73ca2f3dda3 and 758e2d5b1d928f46831f8e38dfaa0e45e3eb72e6